scripts/r.drain/r.drain.py

```diff
@@ -141,10 +140,11 @@ def main():
     atexit.register(cleanup)
 
     if not dirmap:
+        valmap_name = valmap.split("@")[0]
```

Comment:
I suggest adding `maxsplit=1` to clarify what is expected here as input. Or in other words, if we should unify the code for these name-mapset splits, it `maxsplit=1` would be a good way to go.